app/renderer/js/preload.js

```diff
@@ -91,8 +91,15 @@ window.addEventListener('beforeunload', () => {
 
 // electron's globalShortcut can cause unexpected results
 // so adding the reload shortcut in the old-school way
+// Zoom from numpad keys is not supported by electron, so adding it through listeners.
 document.addEventListener('keydown', event => {
 	if (event.code === 'F5') {
```

Comment:
Multiple if else, maybe use switch?

Reply:
@akashnimare I think adding a switch case will make the code more messy as we also have the Ctrl condition to check in zoom options, so we can go with one of these two, although i'd suggest we stay with if else only. 
1.
```
switch (event.code) {
        case 'F5': 
                 ipcRenderer.send('forward-message', 'hard-reload');
                 break;
	case 'NumpadAdd':  
                 if (event.ctrlKey) {
                     ipcRenderer.send('forward-message', 'zoomIn');
                 }
                 break;
        case 'NumpadSubtract':  
                 if (event.ctrlKey) {
                     ipcRenderer.send('forward-message', 'zoomOut');
                 }
                 break;
        case 'Numpad0':  
                 if (event.ctrlKey) {
                     ipcRenderer.send('forward-message', 'zoomActualSize');
                 }
                 break;
```

2.
```
if (event.code == 'F5') {
        ipcRenderer.send('forward-message', 'hard-reload');
} else if (event.ctrlKey) {
        switch (event.code) {
	case 'NumpadAdd':  
                     ipcRenderer.send('forward-message', 'zoomIn');
                     break;
        case 'NumpadSubtract':  
                     ipcRenderer.send('forward-message', 'zoomOut');
                     break;
        case 'Numpad0':
                     ipcRenderer.send('forward-message', 'zoomActualSize');
                     break;
}
```
Which one should we go with ?

Reply:
Okay, if else soundsss good then.